globus_sdk/exc.py

```diff
@@ -133,6 +134,30 @@ def _load_from_json(self, data):
         self.request_id = data["request_id"]
 
 
+class AuthAPIError(GlobusAPIError):
+    """
+    Error class for the API components of Globus Auth.
+
+    Customizes JSON parsing.
+    """
+    def _load_from_json(self, data):
+        """
+        Load error data from a JSON document.
+
+        Sets `code` statically because Auth doesn't have API error codes, and
+        looks for a top-level "error" attribute.
+        """
+        self.code = "Error"
```

Comment:
We do have error codes: https://github.com/globusonline/globus-auth/blob/integration/globus/auth/error/codes.py